Pod/Utilities.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Utilities.swift
+//  ElJayKit
+//
+//  Created by Isaac Greenspan on 12/10/15.
+//  Copyright © 2015 Isaac Greenspan. All rights reserved.
+//
+
+import Alamofire
+
+/// Facilitate generics that Swift doesn't directly allow.
+public struct Of<T> {
```

Comment:
What I really wanted was
```swift
public typealias callback<T> = (Result<T, API.Error>) -> Void
```
but generic bare `typealias`s like that aren't a thing.  The only way I found to do that was to define a generic `struct` that contained the `typealias`.  Naming the struct `Of` seemed the least wrong to me (after going through a couple other options), since it's (theoretically) a container for any generic on one type, it's the member name with that one type—`Of<Foo>.callback` is a callback of `Foo`.  I'm certainly open to naming suggestions.

I had specific `typealias`s everywhere originally, but it felt very repetitive and like it would be a major headache if the error type changed (which actually happened at some point and that may have been the point at which I made the generic) or there were some other overall signature change.